app/src/main/java/net/aiscope/gdd_app/ui/sample_completion/SampleCompletionViewModel.kt

```diff
@@ -147,18 +147,18 @@ class SampleCompletionViewModel @Inject constructor(
         }
     }
 
-    private fun getBloodQuality(bloodQualityValue: String): BloodQuality {
-        return when (bloodQualityValue) {
-            context.getString(R.string.blood_quality_fresh) -> BloodQuality.FRESH
-            context.getString(R.string.blood_quality_old) -> BloodQuality.OLD
-            else -> throw IllegalStateException("$bloodQualityValue blood type is unknown")
+    private fun getSampleAge(sampleAgeValue: String): SampleAge {
+        return when (sampleAgeValue) {
+            context.getString(R.string.sample_age_fresh) -> SampleAge.FRESH
+            context.getString(R.string.sample_age_old) -> SampleAge.OLD
+            else -> throw IllegalStateException("$sampleAgeValue blood type is unknown")

```

Comment:
Exception message to print "sample age".

Reply:
Ah yeah good point. Fixed!